From 7d7a0f386f0efa3d18512d15ddd6aa6d71a81ef8 Mon Sep 17 00:00:00 2001
From: Philippe SWARTVAGHER <phil.swart@gmx.fr>
Date: Sat, 30 Apr 2022 19:16:00 +0200
Subject: [PATCH] Fix quilt configuration

The + sign splitting the QUILT_COLORS string is interpreted by shells
(at least Bash and ZSH) like a (unknown) command.
---
 asciidoc/12-setups.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/asciidoc/12-setups.txt b/asciidoc/12-setups.txt
index 0bf5160..2a390ab 100644
--- a/asciidoc/12-setups.txt
+++ b/asciidoc/12-setups.txt
@@ -88,8 +88,7 @@ if [ -d $d/debian ] && [ -z $QUILT_PATCHES ]; then
     QUILT_PATCH_OPTS="--reject-format=unified"
     QUILT_DIFF_ARGS="-p ab --no-timestamps --no-index --color=auto"
     QUILT_REFRESH_ARGS="-p ab --no-timestamps --no-index"
-    QUILT_COLORS="diff_hdr=1;32:diff_add=1;34:" + \
-            "diff_rem=1;31:diff_hunk=1;33:diff_ctx=35:diff_cctx=33"
+    QUILT_COLORS="diff_hdr=1;32:diff_add=1;34:diff_rem=1;31:diff_hunk=1;33:diff_ctx=35:diff_cctx=33"
     if ! [ -d $d/debian/patches ]; then mkdir $d/debian/patches; fi
 fi
 ----

base-commit: 7cf5b2587bf79036de5c56543df76b5de0965f18
-- 
2.36.0

